docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -1267,34 +1267,73 @@ Return low-level information on the image `name`
     Content-Type: application/json
 
     {
-         "Created": "2013-03-23T22:24:18.818426-07:00",
-         "Container": "3d67245a8d72ecf13f33dffac9f79dcdf70f75acb84d308770391510e0c23ad0",
-         "ContainerConfig":
-                 {
-                         "Hostname": "",
-                         "User": "",
-                         "AttachStdin": false,
-                         "AttachStdout": false,
-                         "AttachStderr": false,
-                         "Tty": true,
-                         "OpenStdin": true,
-                         "StdinOnce": false,
-                         "Env": null,
-                         "Cmd": ["/bin/bash"],
-                         "Dns": null,
-                         "Image": "ubuntu",
-                         "Labels": {
-                             "com.example.vendor": "Acme",
-                             "com.example.license": "GPL",
-                             "com.example.version": "1.0"
-                         },
-                         "Volumes": null,
-                         "VolumesFrom": "",
-                         "WorkingDir": ""
-                 },
-         "Id": "b750fe79269d2ec9a3c593ef05b4332b1d1a02a62b4accb2c21d589ff2f5f2dc",
-         "Parent": "27cf784147099545",
-         "Size": 6824592
+      "ContainerConfig" : {
+        "MacAddress" : "",
+          "Domainname" : "",
+          "StdinOnce" : false,
+          "Entrypoint" : null,
+          "AttachStdin" : false,
+          "WorkingDir" : "",
+          "OnBuild" : null,
+          "AttachStdout" : false,
+          "NetworkDisabled" : false,
+          "OpenStdin" : false,
+          "PortSpecs" : null,
+          "Image" : "37bea4ee0c816e3a3fa025f36127ef8ef0817b3f8fcd7b49eb7b26064f647bb0",
+          "Volumes" : null,
+          "Env" : null,
+          "User" : "",
+          "ExposedPorts" : null,
+          "Labels" : {},
+          "Cmd" : [
+            "/bin/sh",
+            "-c",
+            "#(nop) CMD [\"/bin/bash\"]"
+          ],
+          "Tty" : false,
+          "AttachStderr" : false,
+          "Hostname" : "9ec8c01a6a48"
+      },
+        "VirtualSize" : 188304295,
```

Comment:
Hm, looks like it still doesn't align properly here, also we're losing some example information here (i.e., the image you used doesn't have labels.

Can you build an image, tag that a few times and copy/paste the output of `docker inspect`?

You can use this to build and tag three images with the labels used in the existing documentation?

```bash
docker build -t example:latest - << EOF
FROM ubuntu:latest
LABEL com.example.vendor=Acme com.example.license=GPL com.example.version=1.0
EOF

docker build -t example:1.0 - << EOF
FROM ubuntu:latest
LABEL com.example.vendor=Acme com.example.license=GPL com.example.version=1.0
EOF

docker build -t example:stable - << EOF
FROM ubuntu:latest
LABEL com.example.vendor=Acme com.example.license=GPL com.example.version=1.0
EOF
```

which should give you three images to work with;
```bash
$ docker images example
REPOSITORY          TAG                 IMAGE ID            CREATED              VIRTUAL SIZE
example             1.0                 c40e0290eb61        About a minute ago   188.3 MB
example             latest              c40e0290eb61        About a minute ago   188.3 MB
example             stable              c40e0290eb61        About a minute ago   188.3 MB
```